modules/ROOT/partials/configuration/valid_children.adoc

```diff
@@ -24,7 +24,6 @@ This example shows you how to add `style` as a valid child of `body` and remove
 
 *Type:* `String`
 
-[discrete]
 === Example: Using `valid_children`
 
 [source, js]
```

Comment:
At the end of this file there is a ref that needs to include the full ref since it refers to something in another file:
Line 37: ```xref:schema[`schema`]```